### DISCUSSION TITLE
Powering speakers?

### DISCUSSION THREAD

**Discussion by tiffoknee** [MiczFlor/RPi-Jukebox-RFID#1925] [Q&A]
Are people managing to power the little speakers through the pi? I was hoping to only have one cable coming out, but my pi 3b+ doesn't like it. Suppose i could splice a cable.

**Comment by AlvinSchiller** [comment]
What kind of speaker do you use? And how did you connnect them?

**Comment by Piehti** [comment]
He's properbly talking about the onboard beeper. ^^;
I myself got a small USB Soundbar connected to the Pi. No jack or mini-jack. Just the USB for power and signal.
Works nicely. I think it's a Dell AC511. Not sure.
